frame/benchmarking/src/utils.rs

```diff
@@ -159,6 +158,12 @@ impl From<DispatchError> for BenchmarkError {
 	}
 }
 
+impl From<CodecError> for BenchmarkError {
+	fn from(_: CodecError) -> Self {
+		Self::Stop("Codec error")
+	}
+}
```

Comment:
I slightly worry that the error message will be very vague, not pointing to any specific line, and panicking with an expect would give better error message. maybe it is a better practice not to implement this.

Reply:
I suspected it, so I change it to `expect`. I think some vim macro magic is a bless.